app/views/projects/projects/show.html.erb

```diff
@@ -1,21 +1,85 @@
-<h1><%= @project.title %></h1>
+<div class="row">
+<nav class="projects-nav">
+  <div class="projects-nav__title">
+    <h1><%= @project.title %></h1>
+  </div>
+  <div class="project-nav__menu">
+    <ul class="inline-list">
+      <li>
+        <a href="#">Projects home</a>
+      </li>
+      <li>
+        <a href="#">Tasks</a>
```

Comment:
Where should this link go?

Reply:
It should go to #tasks (on the homepage, adding the ID for the fragment now) - quick Q how can I write the link so that if we're on the homepage already it just outputs `<a href="#tasks">Tasks</a>`, but if not it adds the full path?

Reply:
Just use the full path, as the browser figures out that it doesn't need a full page load.

```erb
<%= link_to 'Tasks', project_path(@project, anchor: 'tasks') %>
```

Reply:
👍 Done